policy/scan/local_scanner.go

```diff
@@ -313,6 +313,8 @@ func (s *LocalScanner) RunAssetJob(job *AssetJob) {
 				log.Error().Err(err).Msgf("failed to synchronize asset to Mondoo Platform %s", job.Asset.Mrn)
 			}
 		}
+		job.ProgressReporter.Score("X")
```

Comment:
This code should be tied to the `job.Reporter.AddScanError(job.Asset, err)` because it always need to go together.

Reply:
Will have a look at it in another PR.

Reply:
Follow-up https://github.com/mondoohq/cnspec/pull/313